wire-ios-sync-engine/Source/UserSession/ZMUserSession.swift

```diff
@@ -467,6 +492,9 @@ public final class ZMUserSession: NSObject {
 
             if e2eiFeature.isEnabled {
                 mlsConversationVerificationManager.startObservingMLSConversationVerificationStatus()
+                self.cRLsDistributionPointsObserver.startObservingNewCRLsDistributionPoints(
+                    from: self.mlsService.onNewCRLsDistributionPoints()
+                )
             }
```

Comment:
We are in the init of `ZMUserSession`. I see this as an anti-pattern to implement side effects like observing and maybe even receiving the first values just when initialising an object. Better would be to have a `setup()` func or similar that is called explicitly.

Nevertheless I understand that this PR is probably not the right place to refactor this..

Reply:
I agree, where would you call that `setup` func from?

Reply:
Let's take it to the refactoring ticket: https://wearezeta.atlassian.net/browse/WPB-6426